### ISSUE TITLE
Original GB DMG, GBP, GBP Light Display Modes Not Showing Pixel Grid

### ISSUE THREAD

**Issue by dji85** [mattpannella/pocket-updater-utility#209]
After enabling all filters the Original GB DMG, GBP, GBP Light do not display the pixel grid effect.

**Comment by Buyukcaglar** [comment]
How this is related to updater utility?

**Comment by dji85** [comment]
@Buyukcaglar how are you verifying it isn’t?

**Comment by Buyukcaglar** [comment]
@dji85 

Pocket firmware download page states exactly the following;
"

Pocket will automatically unlock the CRT Trinitron Original Display Mode **for suitable cores that don't specify any modes. Other modes must be enabled by the core author**

"

Most probably cores that currently do not support these new display modes (hence API's) should be updated by their developers to be able to utilize new  display filters. 

In other words, it may not be possible to force all installed openfpga cores to display/use new filters with a 'magical switch'.

Hence this effectively means that this is not a updater related issue....

**Comment by dji85** [comment]
@Buyukcaglar the other display modes work, including trinatron, game gear, neon pinball matrix, etc. and I’m pretty sure that’s not the meaning of the OS update log.. 

Unless it’s an OS related issue then I’m suggesting it may be related to the updater. Would be helpful if you could verify one way or the other.

**Comment by multi-vitamin** [comment]
https://www.analogue.co/developer/docs/core-definition-files/video-json
it's Core problem

**Comment by mattpannella** [comment]
yeah my app is adding the options to the list but until the core is updated it won't work

**Comment by dji85** [comment]
Thanks @multi-vitamin , that's helpful but if the updater unlocks all display options and there's less than the 16 max total display options available I'm not sure what's happening. Will try hard coding display options for the Spiritualized GB core, thanks!

**Comment by dji85** [comment]
Throws an error when I try modifying the video.json for DMG, @mattpannella you think it's all core related and not Analogue OS that has issues?
